src/_scss/pages/search/filters/otherFilters/_defCheckbox.scss

```diff
@@ -1,5 +1,9 @@
 .def-code-filter {
+    margin-bottom: rem(12);
+    padding: rem(12);
     .def-checkbox-tree {
+      margin-bottom: 2rem;
+      border-bottom: rem(1) solid rgb(214, 215, 217);
```

Comment:
Not sure if design has looked at this, but there is a lot of padding on the left side of the checkbox tree (more so than before and other checkbox tree sections). If we eliminated some of that (right in screenshot below), the labels could fit without scrolling and the second border-bottom may not be necessary.
![defcPadding](https://user-images.githubusercontent.com/7108785/88573576-f793ce80-d00e-11ea-9848-e8cfc880ebe5.png)
<img width="358" alt="checkboxAlignment" src="https://user-images.githubusercontent.com/7108785/88574141-d8497100-d00f-11ea-9b4d-6d68550a01f8.png">

Reply:
@ebdabbs 

Thank you.

Reply:
FWIW I added the border bottom for sake of uniformity of style; specifically when we show a selected pill like this

![image](https://user-images.githubusercontent.com/12897813/88585279-e5229080-d020-11ea-8967-7e9cbe5408ab.png)


Here's NAICS:

![image](https://user-images.githubusercontent.com/12897813/88585298-ece23500-d020-11ea-9f72-f203b7760073.png)